docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc

```diff
@@ -0,0 +1,49 @@
+= Upgrading the build pipeline
+
+To reinforce the security of your application, and to customize the way that {ProductName} builds the components of your application, you must upgrade your build pipeline.
+
+By default, {ProductName} builds the components of applications using, as the name suggests, the default build pipeline. This pipeline offers quick and easy containerized deployment. Default build pipelines also create, for each component, the same three container images as upgraded build pipelines: 
+
+* A business application container image, which runs when your component deploys
+* A signature image, to verify that the business application container image came from a trusted source
+* An attestation image, to verify who built the component, how it was built, and if it was built correctly
+
+However, the default build pipeline skips many tasks that {ProductName} can perform to further secure your application. Again, to perform these checks and scans on any given component, you must upgrade that component’s build pipeline.    
+
+.Prerequisites
+
+* You must have an application that {ProductName} has successfully built and deployed using the default build pipeline.  
+
+.Procedure
+To upgrade the build pipeline:
+
+. In the *Overview* tab of your application, scroll down and select *Manage build pipelines*.
+. If you have not already installed the {ProductName} GitHub application, select Install GitHub application on the “Manage build pipelines” popup page. 
+
++
+[NOTE]
+====
+If you want to restrict the GitHub application’s access to certain repositories only, use the Only select repositories option in GitHub during the installation.
+====
+
+. Returning to the “Manage build pipelines” page in {ProductName}, for any component whose build pipeline you would like to upgrade, select *Send pull request*.  
+. Select *Edit pipeline in GitHub*.
```

Comment:
Currently the button in UI says "Merge in GitHub" - not sure if that's about to change in a future?
![Screenshot 2023-05-05 at 17 48 15](https://user-images.githubusercontent.com/4881144/236506029-c1e6f9e0-79a0-4bc1-a421-7b4e23b1a452.png)